### PR TITLE
Minor fix to service collection extensions

### PR DESCRIPTION
**Pull request by kichalla**
@Eilon Following a pattern here as in other repos

### CODE REVIEW

File: src/Microsoft.AspNetCore.Routing/DependencyInjection/RoutingServiceCollectionExtensions.cs [hunk 1]
```diff
@@ -17,13 +17,11 @@ public static class RoutingServiceCollectionExtensions
     {
         public static IServiceCollection AddRouting(this IServiceCollection services)
```
Comment (pranavkm) [inline src/Microsoft.AspNetCore.Routing/DependencyInjection/RoutingServiceCollectionExtensions.cs]: Docs?

Comment (Eilon) [inline src/Microsoft.AspNetCore.Routing/DependencyInjection/RoutingServiceCollectionExtensions.cs]: Funny, I just logged a bug on that: https://github.com/aspnet/Routing/issues/284

@kichalla I'll assign to you and it's up to you whether to do in this PR or a new one.

**Comment by Eilon** [comment]
The code looks :shipit: to me but would like @rynowak to confirm this makes sense.

**Comment by rynowak** [comment]
These should return `void` - other than that looks fine

**Comment by Eilon** [comment]
@rynowak ah, is `void`-returning part of the pattern? If so, @kichalla @javiercn please be aware of that as you do the cleanup.

**Comment by rynowak** [comment]
Yep. Returning `void` from `AddXyz` methods specifically to remove chaining.

Some methods like `AddMvc()` return their own builder. We want that to stand out.

**Comment by kichalla** [comment]
:up: :date:

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**